### PR TITLE
Update vendoring

### PR DESCRIPTION
**Pull request by SuperQ**
Update vendoring and build.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.0
+0.1.1
```
Comment (brian-brazil) [inline VERSION]: There's never been a release, so this should stay as 0.1.0

Comment (SuperQ) [inline VERSION]: Ok.

**Comment by SuperQ** [comment]
@brian-brazil Removed version change, now it's just a vendoring update.

File: .travis.yml [hunk 2]
```diff
@@ -3,8 +3,10 @@ sudo: false
 language: go
 
 go:
-- 1.5.4
-- 1.6.2
+- 1.8.x
+- tip
```
Comment (AlekSi) [inline .travis.yml]: `tip` is kinda deprecated, it's better to call it `master`: https://github.com/travis-ci/gimme/pull/102

Comment (SuperQ) [inline .travis.yml]: Ahh, this was copy-pasta from other Prometheus use.

**Comment by SuperQ** [comment]
Ok, updated, PTAL.

**Pull request merged**